test/kubernetes-test.sh

```diff
@@ -54,7 +54,7 @@ cd kubernetes
 
 # Kubernetes is only compiling with golang 1.13.4 for versions >=1.17
 sudo rm -fr /usr/local/go
-curl https://dl.google.com/go/go1.13.7.linux-amd64.tar.gz > go1.13.7.linux-amd64.tar.gz
+curl -LO https://dl.google.com/go/go1.13.7.linux-amd64.tar.gz
```

Comment:
just an idea:
what about extracting the version information from `GO_VERSION` and using it here? So, once the officially supported version changes, it doesn't need to be updated manually in everywhere.

Reply:
Good point. I think we can even use it in some other places. But will do that in a separate PR.

Reply:
Sent  #10163 to consume the Go version info from `GO_VERSION` in all places the Go version is used.